README.md

```diff
@@ -554,14 +561,14 @@ PrometheusExporter::Instrumentation::Resque.start
 
 #### Metrics collected by Resque Instrumentation
 
-| Type  | Name                   | Description                            |
-| ---   | ---                    | ---                                    |
-| Gauge | `processed_jobs_total` | Total number of processed Resque jobs  |
-| Gauge | `failed_jobs_total`    | Total number of failed Resque jobs     |
-| Gauge | `pending_jobs_total`   | Total number of pending Resque jobs    |
-| Gauge | `queues_total`         | Total number of Resque queues          |
-| Gauge | `workers_total`        | Total number of Resque workers running |
-| Gauge | `working_total`        | Total number of Resque workers working |
+| Type  | Name             | Description                            |
+| ---   | ---              | ---                                    |
+| Gauge | `processed_jobs` | Total number of processed Resque jobs  |
```

Comment:
qw have to fix these name `resque_processed_jobs` etc.

Reply:
Fixed. https://github.com/discourse/prometheus_exporter/pull/184/commits/ab91609fb46760f4ace046ce68753f379600465a